charts/astronomer/templates/_helpers.yaml

```
@@ -9,6 +9,14 @@
   {{ $globals | toJson | print }}
 {{- end }}
 
+{{ define "defaultXcomISidecarImage" -}}
```

Shouldn't this be `defaultXcomSidecarImage` ?
```suggestion
{{ define "defaultXcomSidecarImage" -}}
```